src/theme/DocItem/Footer/index.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import clsx from "clsx";
+import {ThemeClassNames} from "@docusaurus/theme-common";
+import {useDoc} from "@docusaurus/theme-common/internal";
+import EditThisPage from "@theme/EditThisPage";
+import TagsListInline from "@theme/TagsListInline";
+import styles from "./styles.module.css";
+
+// Eject DocItem/Footer
+
+function CreatedByUser({author}) {
+  return (
+    <>
+      <a href={`https://github.com/${author}`} target="_blank" rel="noreferrer" >
+        Created by <img alt={`${author}`} src={`https://avatars.githubusercontent.com/${author}`} width="20px" style={{borderRadius: "10px", verticalAlign: "sub"}} />  {author}
+      </a>
+    </>
+  );
+}
+
+function TagsRow(props) {
+  return (
+    <div
+      className={clsx(
+        ThemeClassNames.docs.docFooterTagsRow,
+        "row margin-bottom--sm"
+      )}>
+      <div className="col">
+        <TagsListInline {...props} />
+      </div>
+    </div>
+  );
+}
+
+function EditMetaRow({
+  editUrl,
+  author,
+}) {
+  return (
+    <div className={clsx(ThemeClassNames.docs.docFooterEditMetaRow, "row")}>
+      <div className="col">{editUrl && <EditThisPage editUrl={editUrl} />}</div>
+
+      <div className={clsx("col", styles.lastUpdated)}>
+        {(author) && (
+          <CreatedByUser
+            author={author}
+          />
+        )}
+      </div>
+    </div>
+  );
+}
+
+export default function DocItemFooter() {
+  const {metadata} = useDoc();
+  const {editUrl, tags} = metadata;
+  const author = metadata.frontMatter.author || "casdoor";
```

Comment:
use `casdoor` as the default author if the doc does not specify the author.

Reply:
nice